setup.py

```diff
@@ -149,4 +149,5 @@ def get_long_description():
               },
           zip_safe=False,
           test_suite='selftests',
-          python_requires='>=2.6')
+          python_requires='>=2.6',
+          install_requires=['stevedore'])
```

Comment:
To show what I mean, I just installed from your branch into a newly created virtual environment:

    $ pip install -e git://github.com/clebergnu/avocado.git@requirements#egg=avocado-framework
    Obtaining avocado-framework from git+git://github.com/clebergnu/avocado.git@requirements#egg=avocado-framework
      Cloning git://github.com/clebergnu/avocado.git (to requirements) to ./lmr-test-avocado-changes/src/avocado-framework
      Running setup.py egg_info for package avocado-framework
        /usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'use_2to3'
          warnings.warn(msg)
        /usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'python_requires'
          warnings.warn(msg)
        
    Downloading/unpacking stevedore (from avocado-framework)
      Downloading stevedore-1.17.1.tar.gz (501kB): 501kB downloaded
      Running setup.py egg_info for package stevedore
        
        Installed /home/lmr/lmr-test-avocado-changes/build/stevedore/pbr-1.10.0-py2.7.egg
        [pbr] Processing SOURCES.txt
        warning: LocalManifestMaker: standard file '-c' not found
        
        warning: no previously-included files found matching '.gitignore'
        warning: no previously-included files found matching '.gitreview'
        warning: no previously-included files matching '*.pyc' found anywhere in distribution
        warning: no files found matching '*.rst' under directory 'docs'
        warning: no files found matching '*.py' under directory 'docs'
        warning: no files found matching '*.html' under directory 'docs'
        warning: no files found matching '*.css' under directory 'docs'
        warning: no files found matching '*.js' under directory 'docs'
        warning: no files found matching '*.png' under directory 'docs'
        warning: no files found matching '*.txt' under directory 'docs'
        warning: no files found matching '*.jpg' under directory 'docs'
        warning: no files found matching '*.py' under directory 'tests'
    Downloading/unpacking pbr>=1.6 (from stevedore->avocado-framework)
      Downloading pbr-1.10.0.tar.gz (111kB): 111kB downloaded
      Running setup.py egg_info for package pbr
        [pbr] Processing SOURCES.txt
        warning: LocalManifestMaker: standard file '-c' not found
        
        warning: no previously-included files found matching '.gitignore'
        warning: no previously-included files found matching '.gitreview'
        warning: no previously-included files matching '*.pyc' found anywhere in distribution
    Downloading/unpacking six>=1.9.0 (from stevedore->avocado-framework)
      Downloading six-1.10.0.tar.gz
      Running setup.py egg_info for package six
        
        no previously-included directories found matching 'documentation/_build'
    Installing collected packages: avocado-framework, stevedore, pbr, six
      Running setup.py develop for avocado-framework
        /usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'use_2to3'
          warnings.warn(msg)
        /usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'python_requires'
          warnings.warn(msg)
        
        Creating /home/lmr/lmr-test-avocado-changes/lib/python2.7/site-packages/avocado-framework.egg-link (link to .)
        Adding avocado-framework 40.0 to easy-install.pth file
        Installing avocado script to /home/lmr/lmr-test-avocado-changes/bin
        Installing avocado-rest-client script to /home/lmr/lmr-test-avocado-changes/bin
        
        Installed /home/lmr/lmr-test-avocado-changes/src/avocado-framework
      Running setup.py install for stevedore
        [pbr] Generating AUTHORS
        [pbr] AUTHORS complete (0.0s)
        [pbr] Reusing existing SOURCES.txt
      Running setup.py install for pbr
        [pbr] Generating AUTHORS
        [pbr] AUTHORS complete (0.0s)
        [pbr] Reusing existing SOURCES.txt
        Installing pbr script to /home/lmr/lmr-test-avocado-changes/bin
      Running setup.py install for six
        
        no previously-included directories found matching 'documentation/_build'
    Successfully installed avocado-framework stevedore pbr six
    Cleaning up...
    $ avocado --version
    Avocado 40.0
    Failed to load plugin from module "avocado.plugins.remote": ImportError('No module named fabric.exceptions',)
    Failed to load plugin from module "avocado.plugins.vm": ImportError('No module named fabric.exceptions',)
    Failed to load plugin from module "avocado.plugins.docker": ImportError('No module named aexpect',)